memorious/operations/parse.py

```diff
@@ -91,15 +92,52 @@ def parse_for_metadata(context, data, html):
     return data
 
 
+def get_entity_id_from_keys(keys, html):
+    key_string = ""
+    entity_id = ""
+    if isinstance(keys, list):
+        for item in keys:
+            temp_key = html.xpath(item["key"])
+            if len(temp_key) > 0:
+                key_string += "".join(temp_key)
+
+        if (
+            hashlib.md5(key_string.encode("utf-8")).hexdigest()
+            != hashlib.md5("".encode("utf-8")).hexdigest()
+        ):
+            entity_id = hashlib.md5(key_string.encode("utf-8")).hexdigest()
```

Comment:
We can use `make_id` from `memorious.helpers.key` instead of making the key using hashlib.